nuc_morph_analysis/analyses/density/run_all_density_workflows.py

```diff
@@ -0,0 +1,6 @@
+from nuc_morph_analysis.analyses.density import watershed_validate
```

Comment:
Add this (and any others required) to `run_all_manuscript_workflows`?

Reply:
Gonna save this step for the next PR when I make adjustments to the analysis/figure making code. Not all of these will need to go into run all